test/thrift/parser/parser_test.exs

```diff
@@ -111,6 +112,22 @@ defmodule Thrift.Parser.ParserTest do
            ]
   end
 
+  test "parsing cpp_include headers" do
+    cpp_includes =
+      parse(
+        """
+        cpp_include "foo.h"
+        cpp_include "bar.h"
```

Comment:
Perhaps also have a test example that uses slashes, like `”bar/baz.h”`.

Reply:
Added